test/parallel_api/ranges/std_ranges_adjacent_find.pass.cpp

```diff
@@ -22,10 +22,12 @@ main()
     using namespace test_std_ranges;
     namespace dpl_ranges = oneapi::dpl::ranges;
 
-    test_range_algo<0>{big_sz}(dpl_ranges::adjacent_find, std::ranges::adjacent_find, binary_pred);
-    test_range_algo<1>{}(dpl_ranges::adjacent_find, std::ranges::adjacent_find, binary_pred, proj);
-    test_range_algo<2, P2>{}(dpl_ranges::adjacent_find, std::ranges::adjacent_find, binary_pred, &P2::x);
-    test_range_algo<3, P2>{}(dpl_ranges::adjacent_find, std::ranges::adjacent_find, binary_pred, &P2::proj);
+    auto adj_find_checker = _ONEDPL_ALGO(std::ranges::adjacent_find);
```

Comment:
In the past, we have tried to stay away from using internal helpers / utilities / implementation details in our tests.  

If we want to use the same macro here, I think we probably want to re-define it here `test/support/utils.h`, or somewhere similar as `_TEST_PREPARE_CALLABLE`, or something like that, then use that throughout the tests.